docker-compose-integration.yaml

```
@@ -18,16 +18,18 @@ services:
       test: >
         curl --fail -H "authorization: Bearer development" http://localhost:5002/room || exit 1
       interval: 3s
-      retries: 30
+      retries: 10
```

This is still quite high number - with 3s intervals this means the health check will fail after 30 seconds of not responsive server
```suggestion
      retries: 2
```